pyClient/test/test_zeth_constants.py

```diff
@@ -4,39 +4,44 @@
 #
 # SPDX-License-Identifier: LGPL-3.0+
 
-import zeth.utils
-from . import test_utils
+import zeth.encryption
+from zeth import testing_utils
+from zeth.utils import bits_to_bytes_len
 from unittest import TestCase
+from secrets import token_bytes
+from zeth.constants import APK_LENGTH, HIDDEN_VALUE_LENGTH,\
+    RHO_LENGTH, TRAPR_LENGTH
 
 
 class TestZethConstants(TestCase):
 
     def test_encrypt_decrypt(self) -> None:
         """
         Tests the correct encrypt-decrypt flow: decrypt(encrypt(m)) == m
+        where m is 120 bytes long.
```

Comment:
` 120 bytes long` ->  120-byte long

I'd rather write: `where m is encoded on (APK_LENGTH + HIDDEN_VALUE_LENGTH + RHO_LENGTH +TRAPR_LENGTH)/BYTE_LEN bytes`

Reply:
Changed